### PR TITLE
Migrate prefers-reduced-motion article from WebFu

### LINKED ISSUE #1848: content: Migrate prefers-reduced-motion article from WebFu
**Author**: @tomayac 
**Reviewer**: @surma @robdodson @petele 

Migrate https://developers.google.com/web/updates/videos/2019/03/prefers-reduced-motion/ over to https://web.dev/prefers-reduced-motion/

**Target publish date:** before CDS
- [X] Check this box if this is a hard deadline.

- [ ] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #1848

Changes proposed in this pull request:

- Migrate https://developers.google.com/web/updates/videos/2019/03/prefers-reduced-motion/ over to https://web.dev/prefers-reduced-motion/

### CODE REVIEW

**Comment by Malvoz** [comment]
There was a discussion regarding best practices for `prefers-reduced-motion` in https://github.com/mozdevs/cssremedy/issues/11, conclusions resulted in [this CSS snippet](https://github.com/mozdevs/cssremedy/blob/master/css/reminders.css#L31-L48):

> ```CSS
> @media (prefers-reduced-motion: reduce) {
>   *, ::before, ::after {
>     animation-delay: -1s !important;
>     animation-duration: 1s !important;
>     animation-iteration-count: 1 !important;
>     background-attachment: initial !important;
>     scroll-behavior: auto !important;
>     transition-duration: 0s !important;
>   }
> }
> ```

Might be worth to consider... FWIW I have a strong opinion that `scroll-behavior: smooth` should not be allowed when users prefers reduced motion, as it can lead to terrible experiences for users with motion sickness.

**Comment by Malvoz** [comment]
There may be 2 things that were overlooked in that snippet (which I've asked about but haven't gotten any answers yet, see https://github.com/mozdevs/cssremedy/issues/11#issuecomment-557808192):

1. The snippet should probably include `transition-delay: 0s !important;`

2. There's a bug in Safari where negative `animation-delay` is treated as `0s`, resulting in an effective duration of `1s` due to the current `animation-duration` of `1s` instead of the expected `0s`, potential improvement:

```diff
-animation-delay: -1s !important;
-animation-duration: 1s !important;
+animation-delay: -1ms !important;
+animation-duration: 1ms !important;
```

File: src/site/content/en/blog/prefers-reduced-motion/index.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+---
+title: Move Ya! Or maybe, don't, if the user prefers-reduced-motion!
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/en/blog/prefers-reduced-motion/index.md]: Thanks, addressed in https://github.com/GoogleChrome/web.dev/pull/1849/commits/15b72e3fd7d4a468e51c08fb0b4869e920f2ed59.

**Comment by tomayac** [comment]
> There may be 2 things that were overlooked in that snippet, see [mozdevs/cssremedy#11 (comment)](https://github.com/mozdevs/cssremedy/issues/11#issuecomment-557808192):
> 
> 1. The snippet should probably include `transition-delay: 0s !important;`
> 2. There's a bug in Safari where negative `animation-delay` is treated as `0s`, resulting in an effective duration of `1s` due to the current `animation-duration` of `1s` instead of the expected `0s`, potential improvement:
> 
> ```diff
> -animation-delay: -1s !important;
> -animation-duration: 1s !important;
> +animation-delay: -1ms !important;
> +animation-duration: 1ms !important;
> ```

Thanks, addressed in https://github.com/GoogleChrome/web.dev/pull/1849/commits/15b72e3fd7d4a468e51c08fb0b4869e920f2ed59.

File: src/site/content/en/blog/prefers-reduced-motion/index.md [hunk 2]
```diff
@@ -203,10 +203,8 @@ reveal animations are gone, and just the regular scrolling motion is left. The s
 shows the demo in action:
 
 <figure>
-  <video muted playsinline intrinsicsize="1098x870" controls
-    poster="prefers-reduced-motion.png"
-  >
-    <source src="prefers-reduced-motion.mp4" type="video/mp4">
+  <video muted playsinline intrinsicsize="1098x870" controls poster="prefers-reduced-motion.png">
```
Comment (Malvoz) [inline src/site/content/en/blog/prefers-reduced-motion/index.md]: `intrinsicsize` is deprecated (https://github.com/GoogleChrome/web.dev/pull/1937).

File: src/site/content/en/blog/prefers-reduced-motion/index.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+  - thomassteiner
+description: The prefers-reduced-motion media query detects whether the user has requested that the system minimize the amount of animation or motion it uses. This is for users who either require or prefer minimized animations; for example people with vestibular disorders often desire animations to be kept to a minimum.
+date: 2019-03-11
+updated: 2019-12-10
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - media-queries
+  - css
+hero: hero.jpg
+alt: Time lapse of woman in a train
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-reduced-motion/index.md]:
```suggestion
alt: Time-lapse of woman in a train
```

File: src/site/content/en/blog/prefers-reduced-motion/index.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+[demo](https://prefers-reduced-motion.glitch.me). When you scroll down, each HTTP status cat
+alternatingly appears from either the right or the left side. It's a buttery smooth 60 FPS animation,
+but as outlined above, some users may dislike it or even get motion sick by it, so the demo is
+programmed to respect `prefers-reduced-motion`. This even works dynamically, so users can change
+their preference on-the-fly, no reload required. If a user prefers reduced motion, the non-necessary
+reveal animations are gone, and just the regular scrolling motion is left. The screencast below
+shows the demo in action:
+
+<figure>
+  <video muted playsinline intrinsicsize="1098x870" controls poster="prefers-reduced-motion.png">
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-reduced-motion/index.md]:
```suggestion
  <video muted playsinline controls poster="prefers-reduced-motion.png">
```

**Pull request merged**